src/main/webapp/i18n/de/editor.json

```
@@ -31,6 +31,9 @@
             "submitDescription": "Staged, committed, pushed, kompiliert und testet Deine Änderungen.",
             "buildOutput": "Build Ergebnisse",
             "building": "Build und Tests werden ausgeführt...",
+            "queued": "In Warteschlange...",
+            "toolTip": "Abgaben werden zuerst in die Warteschlange gestellt und dann verarbeitet. Die geschätzte Zeit für jeden Schritt wird angezeigt. Bitte beachte, dass dies nur eine Schätzung ist und variieren kann.",
+            "buildFailed": "Build fehlgeschlagen",
```

Biome (1.9.4)

[error] The key buildFailed was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)